shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -624,6 +624,9 @@ static bool ClipRRectContainsPlatformViewBoundingRect(const SkRRect& clip_rrect,
 void FlutterPlatformViewsController::Reset() {
   UIView* flutter_view = flutter_view_.get();
   for (UIView* sub_view in [flutter_view subviews]) {
+    if (![sub_view isKindOfClass:[ChildClippingView class]]) {
```

Comment:
We shy away from RTTI-ish things in the engine. Can we avoid the same in ObjC TUs as well.

Reply:
https://google.github.io/styleguide/cppguide.html#Run-Time_Type_Information__RTTI_ for the reasoning.

Reply:
Good point, I will fix it. Also filed https://github.com/flutter/flutter/issues/126868

Reply:
Interesting read. tho that means we will need to manually keep track of the child clipping views array, and keep that in sync with `UIView::subviews`. not sure if it's worth the effort.

Reply:
>manually keep track of the child clipping views array 

We have it: `active_composition_order_`

Reply:
@chinmaygarde @hellohuanlin Fixed, ptal